h/static/scripts/group-forms/components/EditGroupMembersForm.tsx

```diff
@@ -25,10 +35,22 @@ async function fetchMembers(
     signal,
   });
   return members.map(member => ({
+    userid: member.userid,
     username: member.username,
+    canDelete:
```

Comment:
Maybe `canBeDeleted`? `canDelete` sounds as "this member can delete other members".